assets/blocks/course-list-block/course-list-block-editor.scss

```diff
@@ -0,0 +1,5 @@
+.course-list-block {
```

Comment:
I'd recommend also wrapping this in [`.editor-styles-wrapper`](https://developer.wordpress.org/block-editor/how-to-guides/themes/theme-support/#editor-styles). I'm not sure it's strictly necessary, but it seems safer to add it.

Also, note that I've renamed the `course-list-block` class in #5524.

Reply:
Thanks @donnapep , addressed here https://github.com/Automattic/sensei/pull/5525/commits/cb591dcc2e15804780f3e0b973b17df9eeb52574, can you kindly take a look?